src/registers/model_specific.rs

```diff
@@ -725,4 +753,69 @@ mod x86_64 {
             }
         }
     }
+
+    impl Apic {
+        /// Reads the IA32_APIC_BASE.
```

Comment:
Please change this to either:
```suggestion
        /// Reads the IA32_APIC_BASE MSR.
```
or
```suggestion
        /// Reads IA32_APIC_BASE.
```